### PR TITLE
Fixed cleanup.sh script according to the requirements, squashed to a single commit.

### PR DESCRIPTION
**Pull request by rybalko**
Signed-off-by: Konstantin Rybalko <konstantin.rybalko@gmail.com>

### CODE REVIEW

**Review by petermetz — APPROVED** [review]

**Review by takeutak — APPROVED** [review]

**Comment by takeutak** [comment]
@rybalko Thank you for posting this new pull-request!
There is no problem with the content of the pull-request itself and this was already approved by two maintainers.
But the code checking tool is running now. After that this pull-request can be merged. I'm going to merge this PR after the tool finishes to check.
Could you wait for a while until this PR is merged?

**Comment by takeutak** [comment]
@rybalko The merge was done now. Thanks for your contribution!

**Pull request merged**
